src/pd/protocol.rs

```diff
@@ -124,13 +124,15 @@ impl super::PdClient for RpcClient {
                         region: metapb::Region,
                         leader: metapb::Peer,
                         down_peers: Vec<pdpb::PeerStats>,
-                        pending_peers: Vec<metapb::Peer>)
+                        pending_peers: Vec<metapb::Peer>,
+                        written_bytes: u64)
```

Comment:
I think the function has many args now, maybe we will support keys_read later, so maybe we can use a Options structure.